packages/edit-post/src/components/sidebar/block-sidebar/index.js

```diff
@@ -1,30 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { Panel, PanelBody } from '@wordpress/components';
-import { BlockInspector } from '@wordpress/editor';
-import { __ } from '@wordpress/i18n';
-
-/**
- * Internal Dependencies
- */
-import SettingsHeader from '../settings-header';
-import Sidebar from '../';
-
-const SIDEBAR_NAME = 'edit-post/block';
-
-const BlockSidebar = () => (
-	<Sidebar
-		name={ SIDEBAR_NAME }
-		label={ __( 'Editor settings' ) }
-	>
-		<SettingsHeader sidebarName={ SIDEBAR_NAME } />
-		<Panel>
-			<PanelBody className="edit-post-block-sidebar__panel">
```

Comment:
There's still an occurrence of this now-removed class name in tests:

https://github.com/WordPress/gutenberg/blob/dd7e54bdd4bf15f919eda88b9ff88a836ae88c1a/test/e2e/specs/block-hierarchy-navigation.test.js#L35

Tests are currently running, but if they're passing that would be equally concerning.